static/js/unread_ui.js

```diff
@@ -82,7 +82,7 @@ function consider_bankruptcy() {
     }
 
     var now = new XDate(true).getTime() / 1000;
-    if ((page_params.unread_count > 500) &&
+    if ((page_params.unread_msgs.count > 500) &&
```

Comment:
It definitely doesn't, since that's only present in `page_params`, not in the `/register` API, anyway.